src/data/int/parity.lean

```diff
@@ -47,14 +47,13 @@ begin
   exact @modeq.modeq_add _ _ 1 _ 1 h₁ h₂
 end
 
+@[parity_simps] theorem even_neg {n : ℤ} : even (-n) ↔ even n := by simp [even]
+
 @[simp] theorem not_even_bit1 (n : int) : ¬ even (bit1 n) :=
 by simp [bit1] with parity_simps
 
 @[parity_simps] theorem even_sub {m n : int} : even (m - n) ↔ (even m ↔ even n) :=
-begin
-  conv { to_rhs, rw [←sub_add_cancel m n, even_add] },
-  by_cases h : even n; simp [h]
-end
+by simp with parity_simps
```

Comment:
Why are we even proving this as a `parity_simps` lemma, if `parity_simps` already proves it?

Reply:
Well, it can still be useful to have it available for rewriting. You wouldn't want to interrupt a rw chain just for calling parity_simps.

Reply:
I could still make it a lemma. I guess one argument is if I wanted to `simp [-sub_eq_add_neg] with parity_simps`, then maybe I would still want this to apply. Doesn't make much difference either way

Reply:
Maybe people will want to do `simp only with parity_simps`